smriprep/workflows/anatomical.py

```diff
@@ -27,18 +29,84 @@
     PatchedLTAConvert as LTAConvert,
 )
 from niworkflows.interfaces.images import TemplateDimensions, Conform, ValidateImage
+from niworkflows.interfaces.utils import CopyXForm
+from niworkflows.interfaces.ants import ThresholdImage
 from niworkflows.utils.misc import fix_multi_T1w_source_name, add_suffix
 from niworkflows.anat.ants import init_brain_extraction_wf
 from .norm import init_anat_norm_wf
 from .outputs import init_anat_reports_wf, init_anat_derivatives_wf
 from .surfaces import init_surface_recon_wf
 
+from packaging.version import parse as parseversion, Version
+from warnings import warn
+
+
+def init_n4_only_wf(name='n4_only_wf', omp_nthreads=None):
+    wf = pe.Workflow(name)
+
+    if omp_nthreads is None or omp_nthreads < 1:
+        omp_nthreads = cpu_count()
+
+    inputnode = pe.Node(niu.IdentityInterface(fields=['in_files', 'in_mask']),
+                        name='inputnode')
+
+    outputnode = pe.Node(niu.IdentityInterface(
+        fields=['out_file', 'out_mask', 'bias_corrected', 'bias_image',
+                'out_segm', 'out_tpms']),
+        name='outputnode')
+
+    # Create brain mask (out_mask)
+    thr_brainmask = pe.MapNode(ThresholdImage(
+        dimension=3, th_low=0.00001, th_high=1e10, inside_value=1,
+        outside_value=0), name='thr_brainmask', iterfield=['input_image'])
+
+    # INU correction (output_image -> bias_corrected, out_file)
+    inu_n4 = pe.MapNode(
+        N4BiasFieldCorrection(
+            dimension=3, save_bias=True, copy_header=True,
+            n_iterations=[50] * 5, convergence_threshold=1e-7, shrink_factor=4,
+            bspline_fitting_distance=200),
+        n_procs=omp_nthreads, name='inu_n4_final', iterfield=['input_image'])
+
+    # Tolerate missing ANTs at construction time
+    _ants_version = Registration().version
```

Comment:
I believe this should work out:
```suggestion
    _ants_version = N4BiasFieldCorrection().version
```